src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Main.cs

```diff
@@ -4,18 +4,23 @@
 
 using System;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 using System.Windows;
+using System.Windows.Input;
 using System.Windows.Interop;
 using ManagedCommon;
 using Microsoft.PowerToys.Run.Plugin.System.Win32;
+using Microsoft.PowerToys.Settings.UI.Library;
 using Wox.Infrastructure;
+using Wox.Infrastructure.Storage;
 using Wox.Plugin;
 
 namespace Microsoft.PowerToys.Run.Plugin.System
 {
     public class Main : IPlugin, IPluginI18n
```

Comment:
It should implement `ISettingProvider` so `AdditionalOptions` property make sense

Reply:
Done